package.json

```diff
@@ -72,6 +72,7 @@
     "typescript": "^5.2.2"
   },
   "dependencies": {
+    "@dvsa/cvs-type-definitions": "3.6.0",
```

Comment:
Test result service is now using 4.0.0, are we in a position to bump this too or is it a breaking change?